src/applications/vaos/services/appointment/index.js

```diff
@@ -342,3 +342,28 @@ export function isAtlasLocation(appointment) {
     element => element.tasInfo,
   );
 }
+
+/**
+ * Method to check for the existence of a practitioner
+ * @param {Object} appointment An appointment resource
+ * @return {Boolean} Returns whether or not the appointment has a practitioner.
+ */
+export function hasPractitioner(appointment) {
+  const participant = appointment?.participant;
+  if (participant?.length) {
+    return participant.some(item =>
+      item.actor?.reference?.includes('Practitioner'),
+    );
+  }
+  return false;
+}
+
+/**
+ * Method to parse out the appointment practitioner of participants array
+ * @param {Array} participants An array of appointment participants
+ * @return {Object} Returns the appointment practitioner object.
+ */
+export function getPractitionerDisplay(participants) {
+  return participants.find(p => p.actor.reference.includes('Practitioner'))
```

Comment:
You may want to put optional chaining after `.includes('Practitioner'))` and a fallback because I think if we aren't able to find one, this will throw an error.  e.g.

```jsx
return participants.find(p => p.actor.reference.includes('Practitioner'))?.actor.display || '';
```

Reply:
In addition to this, I think we can potentially have more than one practitioner. So it would probably make sense to do `filter` instead of `find` and return an array from here, then loop through and display each one in the component.

Reply:
Added logic to handle this case 
![image](https://user-images.githubusercontent.com/8315447/95241925-7c078800-07dc-11eb-9cbd-928072e54524.png)